.github/workflows/main.yml

```
@@ -0,0 +1,16 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [$default-branch, develop]
+  pull_request:
+    branches: [$default-branch, develop]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

I think: 

```suggestion
        run: docker build .
```

would be sufficient/same for our purposes here.